src/Styles.js

```diff
@@ -92,6 +103,7 @@ export default css`
   flex-direction: row;
   align-items: center;
   border-bottom: 1px var(--api-parameters-document-title-border-color, #e5e5e5) solid;
+  border: var(--api-parameters-document-title-border);
```

Comment:
@TwoPlusTwoOne, @jarrodek, this property overrides `border-bottom` above and breaks current styling. Please re-review and fix. The same problem here:
https://github.com/advanced-rest-client/api-body-document/pull/21/commits/ded3611524d488d7fc23ab2c12d12a76295a7230#diff-eccc2b8fef024888c50f0586687e68415115c3cc38bd9fda26ba1d4cbd0ce0dcR20
![image](https://user-images.githubusercontent.com/6103913/119276007-0217f800-bc21-11eb-9b3e-8dacf9bfde97.png)

Reply:
@deiteris Thanks for finding this

I've opened the following pull changes to address these issues
- https://github.com/advanced-rest-client/api-method-documentation/pull/43 
- https://github.com/advanced-rest-client/api-body-document/pull/26